alerting/src/main/kotlin/org/opensearch/alerting/MonitorRunner.kt

```diff
@@ -141,4 +146,41 @@ abstract class MonitorRunner {
 
         return NotificationActionConfigs(destination, channel)
     }
+
+    protected fun defaultToPerExecutionAction(
```

Comment:
should we move these to be part of a utility class instead?

Reply:
done